client/signup/steps/import-from/wordpress/import-everything/confirm.tsx

```diff
@@ -0,0 +1,114 @@
+import { Title, SubTitle, NextButton } from '@automattic/onboarding';
+import { sprintf } from '@wordpress/i18n';
+import { Icon, check } from '@wordpress/icons';
+import { useI18n } from '@wordpress/react-i18n';
+import classnames from 'classnames';
+import React, { useState } from 'react';
+import { UrlData } from 'calypso/signup/steps/import/types';
+import { convertToFriendlyWebsiteName } from 'calypso/signup/steps/import/util';
+import ConfirmModal from './confirm-modal';
+import type { SitesItem } from 'calypso/state/selectors/get-sites-items';
+
+import './style.scss';
+
+interface Props {
+	sourceSite: SitesItem | null;
+	sourceSiteUrl: string;
+	sourceUrlAnalyzedData: UrlData;
+	targetSite: SitesItem | null;
+	targetSiteSlug: string;
+	startImport: () => void;
+}
+
+export const Confirm: React.FunctionComponent< Props > = ( props ) => {
+	const { __ } = useI18n();
+
+	/**
+	 ↓ Fields
+	 */
+	const { sourceSiteUrl, sourceUrlAnalyzedData, targetSite, targetSiteSlug, startImport } = props;
+	const [ isModalDetailsOpen, setIsModalDetailsOpen ] = useState( false );
+
+	return (
+		<>
+			<div className={ classnames( 'import__import-everything' ) }>
+				<div className={ classnames( 'import__site-mapper' ) }>
+					<div className={ classnames( 'import-layout', 'import__site-mapper-border' ) }>
+						<div className={ classnames( 'import-layout__column' ) }>
+							<div
+								className={ classnames( 'import__site-mapper-header import__site-mapper-border' ) }
+							>
+								{ __( 'Original site' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **42** times:
`translate( 'Original Site' )` *ES Score: 10*